how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -0,0 +1,163 @@
+# How to use Kata Containers and CRI (containerd plugin)
+
+This document describes how to set up a single-machine Kubernetes cluster. 
+The Kuberentes cluster will use the CRI containerd plugin and Kata Containers to lanuch untrsuted workloads.
+
+## Requirements 
+- Kuberentes, kubelet, kubeadm
+- cri-containerd
+- Kata Containers
+
+For information about the supported version of this components see 
+Kata Containers [versions.yaml](https://github.com/kata-containers/runtime/blob/master/versions.yaml) file.
+
+## Install containerd(with CRI plugin enabled)
+
+Follow the instructions from [CRI installation guide](http://github.com/containerd/cri/blob/master/docs/installation.md)
+
+## Install Kata Containers / Kata
+
+Follow the instructions to install Kata in your system:
```

Comment:
Can you replace this section with a reference to the install README:

- https://github.com/kata-containers/documentation/blob/master/install/README.md